recipes/ansifilter/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "ansifilter" %}
+{% set version = "2.17" %}
+{% set sha256 = "5c8c4d86babbc952141341f70f1ac7a27de5f288b03938b7c8bcb5b84c4217ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://gitlab.com/saalen/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: make && make install PREFIX=$PREFIX
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+test:
+  requires:
+  commands:
+    - ansifilter --version | grep {{ version }}
```

I believe the ``grep`` here is not necessary
```suggestion
    - ansifilter --version
```